src/librustc/diagnostics.rs

```diff
@@ -470,11 +469,13 @@ parameters are involved, this cannot always be done.
 
 So, for example, the following is not allowed:
 
-```compile_fail
+```
```

Comment:
Like said [above](https://github.com/rust-lang/rust/pull/34401/files#diff-48fe837d9fa7d777a951c5853bb98474R463), this error doesn't exist anymore. I didn't remove it in case people would still need it for whatever reason.